lib/pipeline.js

```diff
@@ -147,6 +147,42 @@ class PipelineModel extends BaseModel {
     }
 
     /**
+     * Fetch all secrets that belong to this pipeline
+     * @property secrets
+     * @return Promise
+    */
+    get secrets() {
+        const listConfig = {
+            params: {
+                pipelineId: this.id
+            },
+            paginate: {
+                count: 25,
```

Comment:
Hmmm I can remove this but it looks like dynamo-datastore needs it as required fields:
https://github.com/screwdriver-cd/datastore-dynamodb/blob/master/index.js#L165-L166